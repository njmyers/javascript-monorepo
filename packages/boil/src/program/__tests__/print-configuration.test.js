import printConfiguration from '../print-configuration';

describe('it works', () => {
  test('it is a function ', () => {
    expect(typeof printConfiguration()).toBe('function');
  });

  test('it runs without crashing', () => {
    expect(printConfiguration()).toMatchSnapshot();
  });
});
