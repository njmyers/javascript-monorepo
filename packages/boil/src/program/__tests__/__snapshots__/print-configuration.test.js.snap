// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`program/printConfiguration it runs without crashing 1`] = `undefined`;
